spec/_attachments/ic.did

```diff
@@ -67,6 +67,11 @@ type ecdsa_curve = variant {
     secp256k1;
 };
 
+type schnorr_algorithm = variant {
+    bip340secp256k1;
+    ed25519;
+};
+
```

Comment:
Nit:
```suggestion
type schnorr_algorithm = variant {
    Bip340Secp256K1;
    Ed25519;
};
```

The current version produces these Rust bindings:
```
#[derive(CandidType, Deserialize)]
pub enum SchnorrAlgorithm {
  #[serde(rename="ed25519")]
  Ed25519,
  #[serde(rename="bip340secp256k1")]
  Bip340Secp256K1,
}
```

The capitalized variants would map a bit nicer to Rust because it wouldn't require the `serde(rename)`, and Motoko would be no worse off (but with IMO nicer style)

Reply:
Thanks @sesi200. While your comment makes sense, I'm still somewhat hesitant to apply it because all other variants in `id.did` are also not capitalized, and there is certainly an argument to be made that we should stay consistent with the existing style.

One possible way forward could be to create a separate PR that capitalises all variant tags in `ic.did` and request approval from the interface spec owners. If that PR is approved, I can then adapt the style here accordingly. WDYT?

Reply:
There's an issue with type naming in did file: the interface-spec uses snake_case. Other specs like ICRC-xx and [schnorr_canister](https://github.com/domwoe/schnorr_canister/blob/main/schnorr_canister.did) use CamelCase.

Reply:
Fair enough, consistency is probably better in this case.

We can't just go ahead and capitalize everything as this would be a breaking change

Reply:
@zensh, the API in the [schnorr_canister](https://github.com/domwoe/schnorr_canister/blob/main/schnorr_canister.did) was updated in the meanwhile.